pkgs/tools/networking/subnetcalc/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl, cmake, ninja }:
+
+stdenv.mkDerivation rec {
+  pname = "subnetcalc";
+  version = "2.4.19";
+
+  src = fetchurl {
+    url = "https://www.uni-due.de/~be0001/subnetcalc/download/subnetcalc-${version}.tar.xz";
+    sha256 = "sha256-E/NavAeCx0U9oiYCEo65P6ZFA52SzVqzxSiunmAyzWc=";
+  };
+
+  nativeBuildInputs = [ cmake ninja ];
+
+  meta = with lib; {
+    description = "SubNetCalc is an IPv4/IPv6 subnet address calculator";
+    longDescription = ''
+      SubNetCalc is an IPv4/IPv6 subnet address calculator. For given IPv4 or
+      IPv6 address and netmask or prefix length, it calculates network address,
+      broadcast address, maximum number of hosts and host address range. Also,
+      it prints the addresses in binary format for better understandability.
+      Furthermore, it prints useful information on specific address types (e.g.
+      type, scope, interface ID, etc.).
+    '';
+    homepage = "https://www.uni-due.de/~be0001/subnetcalc/";
+    license = licenses.gpl3Only;
```

```suggestion
    license = licenses.gpl3Plus;
```
It seems to be licensed as GPL3+: https://github.com/dreibh/subnetcalc/blob/master/src/subnetcalc.cc#L8